engine/setup/src/include/display.h

```diff
@@ -39,7 +39,7 @@ void ShowMain();
 
 #define GET_TITLE char title[1000]="Vega Strike Settings"; \
   if (strlen(CONFIG.program_name)+strlen(VERSION)<800) { \
-    sprintf(title, "Settings - %s - Version %s", \
+    snprintf(title, 1000, "Settings - %s - Version %s", \
```

Comment:
per our other discussion - this should be `sizeof(title)-1`

Is this work on top of that branch?

Reply:
thanks!